pages/Transfer.qml

```diff
@@ -782,10 +776,19 @@ Rectangle {
             return false;
         }
 
-        // The transactional information is correct
-        if(!pageRoot.checkInformation(amountLine.text, addressLine.text, paymentIdLine.text, appWindow.persistentSettings.nettype)){
-            if(amountLine.text && addressLine.text)
-                root.sendButtonWarning = qsTr("Transaction information is incorrect.") + translationManager.emptyString;
+        if (addressLine.text == "") {
+            return false;
+        }
+
+        // Address is valid
+        if(addressLine.text && !walletManager.addressValid(addressLine.text, appWindow.persistentSettings.nettype)){
+            root.sendButtonWarning = qsTr("Address is invalid.") + translationManager.emptyString;
+            return false;
+        }
+
+        // Amount is nonzero
+        if (!amountLine.text || parseFloat(amountLine.text) <= 0) {
+            root.sendButtonWarning = qsTr("Enter an amount.") + translationManager.emptyString;
```

Comment:
This error message is never displayed because send button is disabled when amount is empty.